src/blockChainManager/createBlock.ts

```diff
@@ -0,0 +1,7 @@
+import { encodeSHA256 } from './encodeSHA256';
+import { block } from './block';
+
+// eslint-disable-next-line import/prefer-default-export
+export const createBlock = (element, index) => ((index > 0)
```

Comment:
You should create a function named createABlockChain which takes a number as argument and return a blockChain with a length corresponding to that number that we have as an argument @ipsonmoulinga